corehq/apps/userreports/tasks.py

```diff
@@ -48,12 +48,15 @@ def _build_indicators(indicator_config_id, relevant_ids):
             # save is a noop if the filter doesn't match
             adapter.save(doc)
             last_id = doc.get('_id')
-            redis_client.srem(redis_key, last_id)
+            try:
+                redis_client.lrem(redis_key, 1, last_id)
+            except:
+                redis_client.srem(redis_key, last_id)
```

Comment:
I kept support for sets for any process that started with a set in redis. We can remove this support in a week or two once we know there are no processes that got killed with a set in redis instead of the new list